### ISSUE TITLE
Improve support for italics

### ISSUE THREAD

**Issue by apcamargo** [dracula/visual-studio-code#87]
There are many coding fonts such as IBM Plex Mono and Operator Mono that offer true and beautiful italics. It would be nice if Dracula's italics support improved.

- [Dracula fork with improved italics support](https://github.com/shidhincr/vscode-dracula-italics)

Comparision between Dracula and the [Material Theme](https://github.com/equinusocio/vsc-material-theme):

![dracula](https://user-images.githubusercontent.com/22940964/45266858-55799600-b438-11e8-9ca3-992adc55a2e4.png)

![material](https://user-images.githubusercontent.com/22940964/45266859-590d1d00-b438-11e8-8873-9341df2b2200.png)

**Comment by dsifford** [comment]
Italics are currently supported, but just not for the specific scopes material has them defined for. (which specifically, the only real difference I see is that material has `keyword.control` scope italicized which I quite strongly disagree with).

You'll notice also in the screenshots you've provided that function arguments are italicized in dracula but not in material (as well as type annotations and a few other areas that are not pictured).

I'm pretty opinionated as to what should be italicized, and this has been formalized into the [running specification](https://dsifford.github.io/dracula-spec/) for the entire dracula theme.

My opinionatedness on this stems from my belief that any modification to the default text in terms of color _and_ style should be guided by the need to semantically differentiate the text being modified from the rest of the text. 

Taking your screenshot of the material theme above as an example, I have two arguments to make against that:
1. Keywords, statements, and other core language features are fundamental to the code being written. So, in theory, they should not be italicized because by italicizing there is some connotation that there the tokens are "outside of the norm" or non-core language features.
2. If you disagree with argument 1 and think they _should_ be italicized, then you **must** also italicize other language keywords (e.g. `def` in your example) -- Having non-uniform syntax highlighting is worse in my opinion than having no syntax highlighting at all.

So, in short: I appreciate the feedback on this, but I'm going to say no to changing `keyword.control` to italics for the reasons above. Also, because some users don't even want italics at all and this will just compound that issue. (see #78)

Looks like that fork suits your needs. I'd recommend using that.
